packages/pn-commons/src/types/NotificationDetail.ts

```diff
@@ -320,6 +325,12 @@ export enum DigitalDomicileType {
   APPIO = 'APPIO', // PN-2068
 }
 
+export enum DigitalDomicileTypeForCourtesyMessageOnly {
```

Comment:
I would integrate this into interface DomicileType

Reply:
OK, I feared about possible conflict with the manual notification creation about the digital domicile of each recipient, but now I see that the type PEC is hardcoded. Will do as suggested.

Reply:
Done.